templates/case/_reschedule_finish_processing_deadline.html.twig

```diff
@@ -1,23 +1,27 @@
 {% trans_default_domain 'case' %}
 
-<div class="modal fade" id="reschedule_finish_process_deadline{{ case.id }}">
+<div class="modal fade" id="reschedule_finish_process_deadline">
     <div class="modal-dialog">
         <div class="modal-content">
+            {{ form_start(reschedule_form, {'action': path('case_reschedule_finish_processing_deadline', {'id': case.id}), 'attr': {'id': 'reschedule_process_form'}}) }}
             <div class="modal-header">
                 <h5 class="modal-title" id="rescheduleFinishProcessDeadlineModalLabel">{% trans %}Reschedule finish processing deadline{% endtrans %}</h5>
                 <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                     <span aria-hidden="true"><i class="fas fa-times"></i></span>
                 </button>
             </div>
             <div class="modal-body">
-                {{ form_start(reschedule_form, {'action': path('case_reschedule_finish_processing_deadline', {'id': case.id})}) }}
                 {{ form_widget(reschedule_form) }}
             </div>
             <div class="modal-footer">
                 <button type="button" class="btn btn-secondary" data-dismiss="modal">{% trans %}Close{% endtrans %}</button>
-                <button type="submit" class="btn btn-primary">{% trans %}Reschedule{% endtrans %}</button>
-                {{ form_end(reschedule_form) }}
+                <button id="finish_process_submit_button" type="submit" class="btn btn-primary">{% trans %}Reschedule{% endtrans %}</button>
```

Comment:
The id of the button could be a bit more specific. Maybe something like `reschedule_finish_procesing_deadline_submit_button` is better? Although its quite long.

Reply:
Fixed by fe66e0a516dc35b6d759cb463b1b43772aaf1e07.